### PR TITLE
Ssvep tutorial snrtopo

### LINKED ISSUE #3: add snr topography
i have code for a topo plot already.

statistical comparison of topographies would be great, too

### PR DESCRIPTION
**Pull request by dominikwelke**
my first PR for the SNR topoplots. 
just a starter to get the code. 

what exactly we want to show can be decided working on #4 

closes #3

### CODE REVIEW

**Comment by dominikwelke** [comment]
now closes #6 as well

File: tutorials/time-freq/ssvep.py [hunk 1]
```diff
@@ -306,7 +329,7 @@ def snr_spectrum(psds, noise_n_neighborfreqs=1, noise_skip_neighborfreqs=1):
 # - Alternatively, one can subset trials already at the epoch level using MNEs
 #   event information, and create individual PSD and SNR objects.
 #
-# Here we have already subsetted trials before snr calculation (only 12Hz
+# Here we have already sub-setted trials before snr calculation (only 12Hz
```
Comment (kalenkovich) [inline tutorials/time-freq/ssvep.py]:
```suggestion
# Here we have already subsetted trials before snr calculation (only 12Hz
```

Quite sure that is incorrect (see [here](https://en.wiktionary.org/wiki/subset)). Don't care enough to press on it if that is your preferred spelling though.

**Comment by kalenkovich** [comment]
I am sorry my review as of now consists of a single nitpicky grammar correction :) I've been fighting with my new computer to build the docs again. Since this PR is largely adding a new plot, I wanted to make sure that the tutorial builds OK. I should have probably given up and just run the code manually but.. well. At least it seems to work now and I can continue with the review.

**Comment by kalenkovich** [comment]
The fact that it builds implies that your data downloading code works! That is great that you've figured this out.

**Comment by kalenkovich** [comment]
Downloading the dataset throws the following warning:

```
<ipython-input-2-95db52be992b>:39: RuntimeWarning: Setting non-standard config type: "MNE_DATASETS_SSVEP_PATH"
  data_path = mne.datasets.ssvep.data_path()
```

I think we should add this config type to `config.py`. I am attaching a commit patch ([add-config-type1.zip](https://github.com/dominikwelke/mne-python/files/5526847/add-config-type1.zip)) that does exactly that (I did not figure out how to suggest a change in a file that has not been changed in the pull request). If you agree with the suggestion you can use 

```git am add-config-type1.diff```

to add the commit.

**Review by kalenkovich — CHANGES_REQUESTED** [review]
Take a look at the suggested commit in the previous comment. Otherwise, everything is good from my point of view.

**Comment by dominikwelke** [comment]
great!
thanks, i missed the config file. i quckly pushed a commit myself that puts the new line right after the other datasets in the config file

i would merge now. do you have a preference if we do merge commit, squash and merge, or rebase and merge?

also, i didnt find a way to mark your requested change as solved. doenst matter, but do you have an idea how this is done?

**Comment by dominikwelke** [comment]
hmm, with the new line in `utils/config.py` the CI's are unhappy. 
i think it's due to some tests that fail / are not implemented for the dataset, but i dont find it that quickly.. 

maybe we reject the last commit, leave it open for now and figure this out with the more experienced MNE ppl in a final PR.
there is also a subtlety with dataset version that i just hacked/implemented sloppily so far

**Review by kalenkovich — APPROVED** [review]

**Comment by kalenkovich** [comment]
> hmm, with the new line in `utils/config.py` the CI's are unhappy.
> i think it's due to some tests that fail / are not implemented for the dataset, but i dont find it that quickly..
> 
> maybe we reject the last commit, leave it open for now and figure this out with the more experienced MNE ppl in a final PR.
> there is also a subtlety with dataset version that i just hacked/implemented sloppily so far

Let's forget about the config for now then.

**Comment by kalenkovich** [comment]
> i would merge now. do you have a preference if we do merge commit, squash and merge, or rebase and merge?

I prefer non-forward merges because I find it helpful to have the full history. But that is partly because I tend to stay long on a branch before merging. They'll squash it upstream in the end, by the way.
 
> also, i didnt find a way to mark your requested change as solved. doenst matter, but do you have an idea how this is done?

I couldn't figure it out either. I just left another review with the "approve" option this time.

**Comment by dominikwelke** [comment]
ok, the problems for the CI has to be something else.. not sure what, though :/ 

in your new PR #8 it complains as wel without obvious reason

in this case ill restore the config commit and merge the standard merge commit way..

**Pull request merged**